### PR TITLE
added payments via ngenius

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs [hunk 1]
```diff
@@ -1,38 +1,51 @@
...
         {
-            return await CaptureInPayfort()
+            return await Capture()
                 .Tap(WriteAuditLog);
 
-            Task<Result<CreditCardCaptureResult>> CaptureInPayfort() => _payfortService.Capture(request);
+            Task<Result<CreditCardCaptureResult>> Capture()
+            {
+                return paymentProcessor == PaymentProcessors.Payfort 
+                    ? _payfortService.Capture(request)
```
Comment (mingazhev) [inline Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs:45]: Please use switch

File: Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs [hunk 2]
```diff
@@ -55,16 +68,23 @@ Task WriteAuditLog(CreditCardCaptureResult captureResult)
...
             int agentId)
         {
-            return await VoidInPayfort()
+            return await Void()
                 .Tap(WriteAuditLog);
 
-            Task<Result<CreditCardVoidResult>> VoidInPayfort() => _payfortService.Void(request);
+            Task<Result<CreditCardVoidResult>> Void()
+            {
+                return paymentProcessor == PaymentProcessors.Payfort
```
Comment (mingazhev) [inline Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs:83]: and here

File: Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs (hunk 1 above)
Comment (DFSko) [reply Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs]: Fixed

File: Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs (hunk 2 above)
Comment (DFSko) [reply Api/Services/Payments/CreditCards/CreditCardMoneyCaptureService.cs]: Fixed

**Review by mingazhev — APPROVED** [review]

**Pull request merged**